skyportal/tests/api/test_instrument.py

```diff
@@ -31,18 +39,25 @@ def test_token_user_post_get_instrument(super_admin_token):
             'band': 'NIR',
             'filters': ['f110w'],
             'telescope_id': telescope_id,
+            'field_data': pd.read_csv(fielddatafile)[:5].to_dict(orient='list'),
+            'field_region': Regions.read(regionsdatafile).serialize(format='ds9'),
         },
         token=super_admin_token,
     )
     assert status == 200
     assert data['status'] == 'success'
 
+    # wait for the fields to populate
+    time.sleep(15)
+
     instrument_id = data['data']['id']
     status, data = api('GET', f'instrument/{instrument_id}', token=super_admin_token)
     assert status == 200
     assert data['status'] == 'success'
     assert data['data']['band'] == 'NIR'
 
+    assert len(data['data']['fields']) == 5
```

Comment:
Can we easily check the actual field IDs/values against what we expect?

Reply:
@acrellin Added!